src/cli/keepassxc-cli.1

```diff
@@ -104,6 +110,33 @@ with each attribute shown one-per-line in the given order. If no attributes are
 specified, a summary of the default attributes is given.
 
 
+.SS "Diceware options"
+
+.IP "-w, --wordlist <path>"
+Path of the wordlist for the diceware generator. The wordlist must have > 1000 words,
+otherwise the program will fail. If the wordlist has < 4000 words a warning will
+be printed to STDERR.
+
+
+.SS "PassGen options"
+
+.IP "-l"
```

Comment:
@TheZ3ro we should have the same options in the `add` and `edit` commands when generating a new password. That way these options would be standard across the CLI, and this section could be renamed `Password generation options`

Reply:
@TheZ3ro not sure if you want to address this one in this PR, but if not, can you create an issue?